cmd/veneur-emit/main.go

```diff
@@ -242,11 +244,44 @@ func setupSpan(traceID, parentID *int64, name, tags string) (*ssf.SSFSpan, error
 	return span, nil
 }
 
+func streamOutput(wg *sync.WaitGroup, in io.Reader, out io.Writer) {
```

Comment:
I was going to suggest a `TeeReader` here but I'm not sure if that's the right move since it's got it's own feelings on buffering and blocking. But will mention it in case it fits.

Reply:
Not sure that helps us any - the output read from the `TeeReader` would get discarded anyway, so `io.Copy` is pretty much exactly what we want here (: